src/commands/project/init.ts

```diff
@@ -60,7 +72,22 @@ export const initCommandDecorator = (command: commander.Command): void => {
             `--default-ring must be of type 'string', '${defaultRing}' of type '${typeof defaultRing}' given`
           );
         }
-        await initialize(projectPath, { monoRepo, packagesDir, defaultRing });
+
+        if (
+          variableGroupName !== null &&
+          typeof variableGroupName !== "string"
+        ) {
+          throw new Error(
+            `variableGroupName must be of type 'string', ${typeof variableGroupName} given.`
+          );
+        }
+
+        await initialize(projectPath, {
+          defaultRing,
+          monoRepo,
+          packagesDir,
+          variableGroups: variableGroupName === null ? [] : [variableGroupName]
```

Comment:
since there's only one variable group, is it necessary to have `initialize` take an array of variable group names, instead of just one?

Reply:
The pipeline generation in fileutil supports multiple variable groups so the command option can be changed to an array if needed. If we decide to support one variable group, we can refactor the code in spk.